modules/drivers/googleanalytics/src/metabase/driver/googleanalytics/query_processor.clj

```diff
@@ -50,7 +50,9 @@
     (and (= unit :day) (= amount 0))  "today"
     (and (= unit :day) (= amount -1)) "yesterday"
     (and (= unit :day) (< amount -1)) (str (- amount) "daysAgo")
-    :else                             (du/format-date "yyyy-MM-dd" (du/date-trunc unit (du/relative-date unit amount)))))
+    :else                             (du/format-date
+                                        "yyyy-MM-dd"
+                                        (du/date-trunc unit (du/relative-date unit amount)))))
```

Comment:
What is this change for?

Reply:
This was a change that I later reverted but didn’t revert the formatting.